net/DevExtreme.AspNet.Data.Tests/EFSortingTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿using DevExtreme.AspNet.Data.Helpers;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.ComponentModel.DataAnnotations {
+    class KeyAttribute : Attribute { }
+}
```

Comment:
The `KeyAttribute` is internal, and no classes is marked with it at the moment - shouldn't it be public?